fhir-server/src/main/java/com/ibm/fhir/server/resources/Operation.java

```diff
@@ -130,6 +131,49 @@ public Response invoke(@PathParam("operationName") String operationName, Resourc
         }
     }
 
+    @DELETE
+    @Path("${operationName}")
+    public Response invokeDelete(@PathParam("operationName") String operationName) {
+        // This operation is at a top-level, there are no extra values passed through.
+        // This feature is used by operations to execute a specific operation.
```

Comment:
the wording of this comment seemed a bit odd to me...is there a better way to say it?

Reply:
Fixed. in next commit.